charts/karpor/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: karpor
-version: 0.7.3
+version: 0.7.4
 type: application
 appVersion: 0.6.1
```

Comment:
Hi, I have tested and released karpor [v0.6.2](https://github.com/KusionStack/karpor/actions/runs/13212100898) including your new command line parameters, and `appVersion` can be upgraded to `0.6.2` simultaneously :)